File: leetcode/src/test/java/org/luvx/leetcode/java/medium/_19/SolutionTest.java
package org.luvx.leetcode.java.medium._19;

import org.junit.Test;
import org.luvx.leetcode.link.ListNode;
import org.luvx.leetcode.util.LinkUtils;

public class SolutionTest {
    @Test
    public void removeNthFromEndTest() {
        Solution exec = new Solution();
        for (int i = 1; i <= 5; i++) {
            ListNode h = LinkUtils.initLinked(1, 2, 3, 4, 5);
            LinkUtils.printLink(
                    exec.removeNthFromEnd(h, i)
            );
        }
    }
}

File: leetcode/src/test/kotlin/org/luvx/leetcode/java/medium/_916/SolutionTest.kt
package org.luvx.leetcode.java.medium._916;

import org.luvx.leetcode.out
import kotlin.test.Test

class SolutionTest {

    @Test
    fun wordSubsets() {
        val A = arrayOf("amazon", "apple", "facebook", "google", "leetcode")
        val list = listOf(
            // "facebook","google","leetcode"
            // "apple","google","leetcode"
            // "facebook","google"
            // "google","leetcode"
            // "facebook","leetcode"
            arrayOf("e", "o"),
            arrayOf("l", "e"),
            arrayOf("e", "oo"),
            arrayOf("lo", "eo"),
            arrayOf("ec", "oc", "ceo")
        )
        val exec = Solution();
        for (B in list) {
            exec.wordSubsets(A, B).out();
        }
    }
}